### PR TITLE
add d4 files

### PR DESCRIPTION
**Pull request by rannick**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b [THIS-BRANCH-NAME]`

### How to test
- [ ] Do ...

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: .vscode/settings.json
Comment (ivadym) [inline .vscode/settings.json:1]: vscode config file

File: cg_hermes/config/raredisease.py [hunk 1]
```diff
@@ -45,6 +45,12 @@
         "is_mandatory": False,
         "used_by": [UsageTags.SCOUT],
     },
+    frozenset(["qc_bam", "mosdepth_d4"]): {
+        # {"coverage", "d4", "sample_id"}
```
Comment (ivadym) [inline cg_hermes/config/raredisease.py:49]:
```suggestion
```

**Review by ivadym — APPROVED** [review]
Nice ⭐

**Comment by rannick** [comment]
Deployed:
```
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/hiseq.clinical
done.
Log deploy... done.
hermes version: 4.4.8
```

**Pull request merged**